README.md

```diff
@@ -37,6 +38,22 @@ adhocracy+ is designed to make online participation easy and accessible to every
 ### Start a local server
 
     make watch
+    Go to http://localhost:8004/ and login with admin@liqd.net | password
```

Comment:
above line should be outside the code block, see original:
![image](https://github.com/user-attachments/assets/2e44de86-2bdb-4245-aa89-6d3df44bc8a0)